### DISCUSSION TITLE
Self host source code

### DISCUSSION THREAD

**Discussion by ghost** [majestrate/llarp#20] [General]
Move source code to self hosted git such as gitea once llarp goes mainstream

**Comment by majestrate** [comment]
> Move source code to self hosted git such as gitea once llarp goes mainstream

to expand on this, i want to not rely on github. right now i am using the network effect for contributions and such and once there is enough momentum we'll move things to a gitea that is not hosted by me. i dont want to be the central point of failure for infra.
